lib/docs/filters/ember/entries.rb

```diff
@@ -2,58 +2,39 @@ module Docs
   class Ember
     class EntriesFilter < Docs::EntriesFilter
       def get_name
-        if base_url.path.start_with?('/api')
-          name = at_css('h1').child.content.strip
-          # Remove "Ember." prefix if the next character is uppercase
-          name.sub! %r{\AEmber\.([A-Z])(?!EATURES)}, '\1'
+        name = at_css('h1').content
+        if base_url.host.start_with?('api')
+          name.gsub!('Package', '')
+          name.gsub!('Class', '')
+          name.strip!
           name << ' (methods)' if subpath.end_with?('/methods')
           name << ' (properties)' if subpath.end_with?('/properties')
           name << ' (events)' if subpath.end_with?('/events')
-          name
-        else
-          name = at_css('article h1').content.remove('Edit Page').strip
-          name = at_css('li.toc-level-0.selected > a').content if name == 'Introduction'
-          name
         end
+        name
       end
 
       def get_type
-        if base_url.path.start_with?('/api')
+        if base_url.host.start_with?('api')
           name = self.name.remove(/ \(.*/)
-          if name =~ /\A[a-z\-]+\z/
-            'Modules'
-          elsif name.start_with?('DS')
-            'Data'
-          elsif name.start_with?('RSVP')
-            'RSVP'
-          elsif name.start_with?('Test')
-            'Test'
-          elsif name.start_with?('Ember')
-            name.split('.')[0..1].join('.')
+          if name.start_with?('@') || name == 'rsvp'
+            'Packages'
           else
-            name.split('.').first
+            'Classes'
           end
         else
-          if node = at_css('li.toc-level-0.selected > a')
-            "Guide: #{node.content.strip}"
-          else
-            'Guide'
-          end
+          'Guide'
         end
       end
 
       def additional_entries
-        return [] unless base_url.path.start_with?('/api')
+        return [] unless base_url.host.start_with?('api')
 
         css('section').each_with_object [] do |node, entries|
           next unless heading = node.at_css('h3[data-anchor]')
           next if node.at_css('.github-link').content.include?('Inherited')
-          name = heading.at_css('span').content.strip
 
-          # Give their own type to "Ember.platform", "Ember.run", etc.
-          if self.type != 'Data' && name.include?('.')
-            type = "#{self.name.remove(/ \(.*/)}.#{name.split('.').first}"
-          end
+          name = heading.at_css('span').content.strip
```

Comment:
This is the list I get when searching for "checkbox". Only 3 out of x properties are shown. The entries Checkbox `(events)`, `.` and `.()` all link to /ember/classes/checkbox/methods.

![Screenshot_2021-02-19 Ember js Checkbox — DevDocs](https://user-images.githubusercontent.com/782446/108562602-682e6f80-7300-11eb-9a0f-292f364cd272.png)

Reply:
The reason that this is only listing items that do not have Inherited in one of the documentation fields (it's actually in the line above this one.

`          next if node.at_css('.github-link').content.include?('Inherited')`

I can get rid of that rule if that helps. I'll investigate the weird `.` and `.()` entries.

Reply:
Ok I fixed the `Checkbox.` issue, however, it seems like `Checkbox.()` is a bug with the Ember docs themselves. This entry is missing a name https://api.emberjs.com/ember/3.25/classes/Checkbox/methods?anchor= and is thereby coming out like that. I think I'd rather fix it in the Ember docs themselves than have a special case for this in the scraper.